itest/tapd_harness.go

```diff
@@ -257,7 +257,8 @@ func newTapdHarness(t *testing.T, ht *harnessTest, cfg tapdConfig,
 		BackoffCfg:         &hashmailBackoffCfg,
 	}
 	finalCfg.UniverseRpcCourier = &proof.UniverseRpcCourierCfg{
-		BackoffCfg: &universeRpcBackoffCfg,
+		BackoffCfg:            &universeRpcBackoffCfg,
+		ServiceRequestTimeout: 2 * time.Second,
```

Comment:
If I understand the changes in this PR correctly, _this_ timeout should actually trigger (instead of the 5s in the previous commit) if the hashmail server isn't running or responding. So perhaps we can also get some of the "lost" seconds back by using a lower value here?

Reply:
I think I've managed to lower this significantly in the latest commits.